Mod no longer loads

Here's launch log.
```
( -1 FPS)		ResonitePlatformSpoof
( -1 FPS)	[ERROR] [MonkeyLoader|ResonitePlatformSpoof] Error while intitializing RML Mod ResonitePlatformSpoof:
HarmonyLib.HarmonyException: Patching exception in method null
   at void HarmonyLib.PatchClassProcessor.ReportException(Exception exception, MethodBase original)
   at List<MethodInfo> HarmonyLib.PatchClassProcessor.Patch()
   at void HarmonyLib.Harmony.PatchAll()+(Type type) => { }
   at void HarmonyLib.CollectionExtensions.Do<Type>(IEnumerable<Type> sequence, Action<Type> action)
   at void HarmonyLib.Harmony.PatchAll(Assembly assembly) x 2
   at void ResonitePlatformSpoof.ResonitePlatformSpoof.OnEngineInit()
   at bool ResoniteModLoader.ResoniteMod.Run() ---> System.ArgumentException: Undefined target method for patch method static System.Void ResonitePlatformSpoof.ClientPatch::Prefix(FrooxEngine.SyncMessage& message)
   at List<MethodInfo> HarmonyLib.PatchClassProcessor.PatchWithAttributes(ref MethodBase lastOriginal, bool unpatch)
   at List<MethodInfo> HarmonyLib.PatchClassProcessor.Patch()
   --- End of inner exception stack trace ---

   at void Elements.Core.UniLog.Error(string message, bool stackTrace)
   at void MonkeyLoader.Resonite.ResoniteLoggingHandler.Error(Func<object> messageProducer)
   at void MonkeyLoader.Logging.MulticastLoggingHandler.Error(Func<object> messageProducer)
   at void MonkeyLoader.Logging.LoggingController.LogInternal(LoggingLevel level, string identifier, Func<object> messageProducer)
   at void MonkeyLoader.Logging.Logger.Error(Func<object> messageProducer)
   at bool ResoniteModLoader.ResoniteMod.Run()
   at void MonkeyLoader.MonkeyLoader.RunMonkeys(IEnumerable<Mod> mods) x 2
   at void MonkeyLoader.MonkeyLoader.RunMods(params Mod[] mods)
   at Task ResoniteModLoader.ModLoader.InitializeFrooxEnginePostfixAsync(Task __result)+() => { }
   at void System.Threading.Tasks.Task.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at bool System.Threading.ThreadPoolWorkQueue.Dispatch()
   at bool System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()
```

--- a/ResonitePlatformSpoof/ResonitePlatformSpoof.cs
+++ b/ResonitePlatformSpoof/ResonitePlatformSpoof.cs
@@ -46,7 +46,7 @@ namespace ResonitePlatformSpoof
             Msg("Hooks installed successfully!");
         }
 
-        [HarmonyPatch(typeof(Session), nameof(Session.EnqueueForTransmission), new Type[] { typeof(SyncMessage), typeof(bool) })]
+        [HarmonyPatch(typeof(SessionOutgoingMessageManager), "EnqueueForTransmission", new Type[] { typeof(SyncMessage), typeof(bool), typeof(bool) })]
         private static class ClientPatch
         {
             private static void Prefix(ref SyncMessage message)

